runtime/wasm/src/module/mod.rs

```diff
@@ -76,7 +76,7 @@ impl ToAscPtr for subgraph::TriggerData {
         heap: &mut H,
         gas: &GasCounter,
     ) -> Result<AscPtr<()>, HostExportError> {
-        asc_new(heap, &self.entity.sorted_ref(), gas).map(|ptr| ptr.erase())
+        asc_new(heap, &self.entity, gas).map(|ptr| ptr.erase())
```

Comment:
Is sorting no longer needed? Will that break existing behavior ?

Reply:
or both

Reply:
it is indeed the only entry point, calling `to_asc_ptr` on `subgraph::TriggerData` is the only way in which the mapping is going to get the entity trigger, which internally calls the `EntityWithType.to_asc_obj` which gets the sorted_ref() of the entity as you mentioned happens in `external.rs` 

Hope that answers that.